convert2rhel/toolopts.py

```diff
@@ -87,13 +102,22 @@ def __init__(self):
         self.els = False
         self.activity = None
 
+        # Settings
+        self.incomplete_rollback = None
+        self.tainted_kernel_module_check_skip = None
+        self.outdated_package_check_skip = None
+        self.allow_older_version = None
+        self.allow_unavailable_kmods = None
+        self.configure_host_metering = None
+        self.skip_kernel_currency_check = None
```

Comment:
I'd say apply at class level instead of inside init. Then add stuff like this (haven't verified what it should be)
```suggestion
        # Settings
        self.incomplete_rollback = None  # type: List[str]|None
        self.tainted_kernel_module_check_skip = None  # type: str|None
        self.outdated_package_check_skip = None  # type: str|None
        self.allow_older_version = None  # type: str|None
        self.allow_unavailable_kmods = None  # type: str|None
        self.configure_host_metering = None#   type: str|None
        self.skip_kernel_currency_check = None  # type: str|None
```

Reply:
We can't initialize them at that moment. Our code does not handle that very well right now, so it is better to have then as `None` instead of `0`. 

https://github.com/oamg/convert2rhel/pull/1285/files this would be the next step to get them initialized and properly checked

Reply:
Regarding moving them to be a class level instead of under init, what would be the advantage of that? I'm personally not much a fan of that approach, but maybe there is a better reason to that and I'm not aware of.

Reply:
Added type comments